site/src/sphinx/auth.rst

```diff
@@ -0,0 +1,133 @@
+
+- ``Owner`` of a project
+
+  - the administrator of a project. The owners can create a new repository and remove a repository from the project.
+    They are also able to remove the project from the system and configure permissions for role, member and token.
+    As you expected, the administrators also can do that because they have every permission of the system.
+
+- ``Member`` of a project
+
+  - the user who can see the configuration of a project, but the members is not allowed to change the configuration.
+
+- ``Guest`` of a project
+
+  - the user who is logged in but he or she is neither an owner nor a member of a project. The user is not one of
+    the administrators, too. The guest is not allowed to see the configuration of a project.
+
+.. note::
+
+    Do not forget to make a new ``Application Token`` before adding a token to a project. ``Add a token`` button
+    would be disabled if there is no token.
+
+You can see the configuration UI for a repository when you click the name of repository in the ``Role Permission`` list.
+The following image shows the configuration of the ``main`` repository.
+With this UI, you can do the followings.
+
+- Changing the role of a member or a token in the project
+- Setting permissions of each role for a repository
+- Setting permissions of the specific member or token for a repository
+
+.. image:: _images/auth_3.png
+
+Permissions can be specified for a repository only. So a user can configure their repositories with
+the different access control levels. There are only two types of the permission now, which are ``READ`` and ``WRITE``.
+``WRITE`` permission automatically includes ``READ`` permission, so you cannot give ``WRITE`` permission only
+to an user, a token or any role.
+
+Every access of HTTP API will be controlled by the access control system. A request is allowed only if the
+user of the request has the right to do the action. If permissions for the user are specified in the repository
+configuration, it would be used first to control the request. If it does not exist, permissions for each role of
+the repository would be used to do that.
+
+Application Token
+^^^^^^^^^^^^^^^^^
+
+When a user uses the web UI, he or she should login first to get a token for the web session. But what should a user
+do when he or she uses one of *Central Dogma* clients? The user may login *Central Dogma* server through HTTP login API
+and gets a session token. But it is inconvenient and the user may write more complicated codes because
+the user should log in again when the session token is expired. ``Application Token`` would be useful for this case.
+
+``Application Token`` is like a virtual user, so it can have any role in a project. Also, its permissions can be specified
+in a repository configuration like a member. To get a new token, a user can use ``Application Tokens`` menu of the web UI.
+``Application ID`` is should be unique to identify where the request is from.
```

Comment:
- is should be -> has to be
- the request is from -> a client request comes from